### PR TITLE
SOAPUIOS-375 Fix progress bar update in parallel mode

### PR DESCRIPTION
**Pull request by DmitryNAleshin**
https://smartbear.atlassian.net/browse/SOAPUIOS-375

### CODE REVIEW

File: soapui/src/main/java/com/eviware/soapui/impl/wsdl/panels/project/WsdlProjectTestSuitesTabPanel.java [hunk 1]
```diff
@@ -371,7 +374,14 @@ public void afterTestSuite(ProjectRunner testScenarioRunner, ProjectRunContext r
                 return;
             }
 
-            progressBar.setValue(++finishCount);
+            finishCount.incrementAndGet();
+            Platform.runLater(new Runnable() {
+                @Override
+                public void run() {
+                    progressBar.setValue(finishCount.get());
+                    System.out.println("setting coutn to " + String.valueOf(finishCount.get()));
```
Comment (Alexander-Sviridov) [inline soapui/src/main/java/com/eviware/soapui/impl/wsdl/panels/project/WsdlProjectTestSuitesTabPanel.java]: We need to remove ``System.out.println``.

**Review by Alexander-Sviridov — CHANGES_REQUESTED** [review]

**Review by Alexander-Sviridov — APPROVED** [review]

**Comment by jpraet** [comment]
I think this caused a bug in SoapUI 5.5: https://community.smartbear.com/t5/SoapUI-Open-Source/SoapUI-5-5-0-will-not-run-project-or-suites-Toolkit-not/td-p/179525

Platform.runLater throws "java.lang.IllegalStateException: Toolkit not initialized".

**Comment by DmitryNAleshin** [comment]
Not reproduced for me 
image

**Comment by jpraet** [comment]
It fails for me if I disable showing the Endpoint explorer and the startup page.

> Tue Mar 05 19:36:57 CET 2019:ERROR:Exception during Test Execution
   java.lang.IllegalStateException: Toolkit not initialized
   	at com.sun.javafx.application.PlatformImpl.runLater(PlatformImpl.java:273)
   	at com.sun.javafx.application.PlatformImpl.runLater(PlatformImpl.java:268)
   	at javafx.application.Platform.runLater(Platform.java:83)
   	at com.eviware.soapui.impl.wsdl.panels.testsuite.WsdlTestSuiteDesktopPanel$InternalTestSuiteRunListener.afterTestCase(WsdlTestSuiteDesktopPanel.java:430)
   	at com.eviware.soapui.impl.wsdl.testcase.WsdlTestSuiteRunner.notifyAfterRunTestCase(WsdlTestSuiteRunner.java:200)
   	at com.eviware.soapui.impl.wsdl.testcase.WsdlTestSuiteRunner.runSequential(WsdlTestSuiteRunner.java:130)
   	at com.eviware.soapui.impl.wsdl.testcase.WsdlTestSuiteRunner.internalRun(WsdlTestSuiteRunner.java:94)
   	at com.eviware.soapui.impl.wsdl.testcase.WsdlTestSuiteRunner.internalRun(WsdlTestSuiteRunner.java:46)
   	at com.eviware.soapui.impl.wsdl.support.AbstractTestRunner.run(AbstractTestRunner.java:128)
   	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
   	at java.util.concurrent.FutureTask.run(Unknown Source)
   	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
   	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
   	at java.lang.Thread.run(Unknown Source)

**Pull request merged**